add runtime configurability in getting started (#8197)
cloudflare/cloudflare-docs: content/pages/platform/functions/get-started.md

Refer to [Routing](/pages/platform/functions/routing/) for more information on route customization.

### Runtime Features

> Workers runtime features such as Node.js compatibility can be enabled by configuring Pages Functions settings such as [compatibility date and compatibility flag](/workers/platform/compatibility-dates/). These can be configured for local development [in Wrangler](/workers/wrangler/commands/#dev-1) and in the dashboard. 

Reviewer: ```suggestion
Workers runtime features, including compatibility with a subset of Node.js APIs, setting a [compatibility date or compatibility flag](/workers/platform/compatibility-dates/). With Pages Functions, you can configure these by passing an argument to your [Wrangler](/workers/wrangler/commands/#dev-1) command or by setting them in the dashboard. (Project Settings > Functions > Compatibility Flags)
```

> Additionally, you can use other Cloudflare products from within your Pages project by [configuring bindings](/pages/platform/functions/bindings/). 

Reviewer: ```suggestion
Additionally, you can use other Cloudflare products from within your Pages project by configuring [bindings](/pages/platform/functions/bindings/). 
```

- Customize your [Function's routing](/pages/platform/functions/routing/)
- Review the [API reference](/pages/platform/functions/api-reference/)

> - Learn how to [debug your Function](/pages/platform/functions/debugging-and-logging/)

Reviewer: ```suggestion
- Learn how to [debug your Function](/pages/platform/functions/debugging-and-logging/)
```
